scripts/build-desktop.sh

```diff
@@ -312,28 +339,32 @@ function bundleLinux() {
 
   rm -f Application-x86_64.AppImage Status-x86_64.AppImage
 
-  [ $VERBOSE_LEVEL -ge 1 ] && ldd $(joinExistingPath "$usrBinPath" 'Status') 
+  [ $VERBOSE_LEVEL -ge 1 ] && ldd $(joinExistingPath "$usrBinPath" 'Status')
   desktopFilePath="$(joinExistingPath "$WORKFOLDER" 'AppDir/usr/share/applications/Status.desktop')"
   pushd $WORKFOLDER
-    [ $VERBOSE_LEVEL -ge 1 ] && ldd $usrBinPath/Status
     cp -r assets/share/assets $usrBinPath
     cp -rf StatusImAppImage/* $usrBinPath
     rm -f $usrBinPath/Status.AppImage
   popd
 
+  # Ensure the binary isn't using the interpreter in Nix's store
+  patchelf --set-interpreter /lib64/ld-linux-x86-64.so.2 "$usrBinPath/Status"
+  # Tell linuxdeployqt about all the different lib folders in Nix's store 
+  local all_deps=($(get_recursive_dependencies "$usrBinPath/Status"))
```

Comment:
Clever use of `ldd` to find necessary `LD_LIBRARY_PATH` contents to compile them in with `deployqt`. Very cool!